contracts/fee/IFeedAddress.sol

```diff
@@ -0,0 +1,8 @@
+// SPDX-License-Identifier: BUSL-1.1
```

Comment:
this should not be needed, the Chainlink logic is already in https://github.com/gmx-io/gmx-synthetics/blob/main/contracts/oracle/ChainlinkPriceFeedUtils.sol#L62

Reply:
As per the above comment, this is no longer needed and has been removed.